cypress/integration/seededFlows/articleFlows/postModerationTools.spec.js

```diff
@@ -151,22 +151,154 @@ describe('Moderation Tools for Posts', () => {
         });
       });
     });
+
+    context('when suspending user', () => {
+      beforeEach(() => {
+        cy.get('@moderatorUser').then((user) => {
+          cy.loginAndVisit(user, '/series_user/series-test-article-slug');
+          cy.findByRole('heading', { level: 1, name: 'Series test article' });
+          cy.findByRole('button', { name: 'Moderation' }).click();
+        });
+      });
+
```

Comment:
What are your thoughts on not having a dedicated test for if the button exists?

I ask, because we're interacting with the button later on (e.g. in the `it('should not suspend the user when no reason given'` and `it('should suspend the user when reason given'`).  By not having the explicit test we'd reduce the computational cost of the test suite (e.g. don't spin up another relatively expensive integration test).